src/sparseml/pytorch/sparsification/quantization/helpers.py

```diff
@@ -711,15 +708,20 @@ def fuse_module_conv_bn_relus(
     if len(current_block) > 1:
         conv_blocks.append(current_block)
     if conv_blocks:
-        # manually save and move hooks surrounding fused blocks into new fused modules
-        # due to torch.quantization error when a module has more than one hook
-        block_hooks = _delete_get_block_hooks(module, conv_blocks)
-
         # run torch fusion
-        torch_quantization.fuse_modules(module, conv_blocks, inplace=True)
+        if _PARSED_TORCH_VERSION < version.parse("1.10.0"):
+            # manually save and move hooks surrounding fused blocks
+            # into new fused modules due to torch.quantization
+            # error when a module has more than one hook
+            block_hooks = _delete_get_block_hooks(module, conv_blocks)
```

Comment:
we'll need to keep this in the 1.12 call :/ 

https://github.com/pytorch/pytorch/blob/82dff8ee09278bfea385c27d21d88b978ef911c9/torch/ao/quantization/fuse_modules.py#L61